2025.md

```diff
@@ -5,7 +5,192 @@ permalink: /index.html
 
 # Lucca-Medievale-2025
 
-## Lucca Medievale 2025
+## 🗣️ **Lingua**
+
+* [Italiano 🇮🇹](https://luccamedievale.it/)
+* [English 🇺🇸 🇬🇧](https://luccamedievale-it.translate.goog/?_x_tr_sl=it&_x_tr_tl=en&_x_tr_hl=it&_x_tr_pto=wapp)
```

Comment:
comunque google translate era veramente l'alternativa low-effort per la versione inglese, oggi potresti facilmente farti fare una traduzione al volo (mantenendo la formattazione markdown) e buttarla in una pagina separata (quindi staccandosi da google translate che alla fine non ti permette di controllare l'output)